packages/react/src/components/Slider/Slider.stories.js

```diff
@@ -53,6 +53,41 @@ export const SliderWithHiddenInputs = () => (
   />
 );
 
+export const SliderWithHiddenInputsAndFormatLabel = () => (
+  <Slider
+    labelText="Slider label with percentage"
+    value={50}
+    min={0}
+    max={100}
+    stepMultiplier={10}
+    step={1}
+    noValidate
+    hideTextInput
+    formatLabel={(val) => `${val}%`}
+  />
+);
```

Comment:
```suggestion
```
I'm not sure if we should keep this story. Maybe we can show in the `fomartLabel` [description here](https://github.com/carbon-design-system/carbon/blob/a4f9f5584b1ae1bd043d27bbf32a84ff73750631/packages/react/src/components/Slider/Slider.tsx#L344-L347) how to set the `formatLabel` like that: 
```js
formatLabel={(val) => `${val}%`} 
```
We already have the `SliderWithCustomValueLabel` as an example also.